sympy/vector/integrals.py

```diff
@@ -0,0 +1,30 @@
+from sympy.vector.coordsysrect import CoordSys3D
+from sympy.integrals import Integral
+
+class ParametricIntegral(Integral):
```

Comment:
`Integral` inherits associative operator, which itself overloads the `__new__` constructor.